### PR TITLE
Add AddPermission to AuthManager

### PR DESCRIPTION
**Pull request by boy-uber**

### CODE REVIEW

File: common/auth_util.go [hunk 1]
```diff
@@ -67,6 +68,25 @@ func GetResourceURNCreateConsumerGroup(scommon SCommon, cgPath *string) string {
 	return fmt.Sprintf(resourceURNTemplateCreateConsumerGroup, getTenancyLowerCase(scommon), strings.ToLower(cgPathString))
 }
 
+// GetResourceURNOperateConsumerGroup returns the resource URN to operate consumer group (read, delete), e.g. urn:cherami:dst:zone1_prod:/cg_prefix/cg1
```
Comment (datoug) [inline common/auth_util.go]: update comment: urn:cherami:cg:zone1_prod

File: services/frontendhost/frontend.go [hunk 2]
```diff
@@ -647,6 +647,12 @@ func (h *Frontend) CreateDestination(ctx thrift.Context, createRequest *c.Create
 		return nil, err
 	}
 
+	dstResource := common.GetResourceURNOperateDestination(h.SCommon, createRequest.Path)
```
Comment (datoug) [inline services/frontendhost/frontend.go]: can you add a comment regarding this section?

File: services/frontendhost/frontend.go [hunk 3]
```diff
@@ -1695,8 +1715,25 @@ func (h *Frontend) checkAuth(ctx thrift.Context, authResource string, operation
...
+func (h *Frontend) addPermissions(authSubject common.Subject, authResource string, operations []common.Operation, logger bark.Logger) error {
+	for _, operation := range operations {
+		err := h.GetAuthManager().AddPermission(authSubject, operation, common.Resource(authResource))
+		if err != nil {
+			logger.WithFields(bark.Fields{
+				common.TagErr:       err,
+				common.TagSubject:   authSubject,
+				common.TagResource:  authResource,
+				common.TagOperation: operation,
+			}).Info("Failed to add permission")
```
Comment (datoug) [inline services/frontendhost/frontend.go]: change this to error log?

File: common/auth_util.go (hunk 1 above)
Comment (boy-uber) [inline common/auth_util.go]: Good catch!

File: services/frontendhost/frontend.go (hunk 3 above)
Comment (boy-uber) [inline services/frontendhost/frontend.go]: Yes

File: services/frontendhost/frontend.go (hunk 2 above)
Comment (boy-uber) [inline services/frontendhost/frontend.go]: Yes

**Pull request merged**